sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyCreateOptions.cs

```diff
@@ -13,6 +13,6 @@ public class KeyCreateOptions
         public DateTimeOffset? NotBefore { get; set; }
         public DateTimeOffset? Expires { get; set; }
         public bool? Enabled { get; set; }
-        public IDictionary<string, string> Tags { get; set; }
+        public IDictionary<string, string> Tags { get; private set; } = new Dictionary<string, string> ();
```

Comment:
>public IDictionary<string, string> Tags { get; private set; } = new Dictionary<string, string> (); [](start = 8, length = 98)

What's the motivation for making Tags not settable?

Reply:
Just when initializing it. Once Keys is created you can modify the content of the tags as you want.